features/landscape.feature

```diff
@@ -0,0 +1,285 @@
+        """
+        When I run `pro status` with sudo
+        Then stdout matches regexp:
+        """
+        landscape +yes +warning
+        """
+        Then stdout contains substring:
+        """
+        Landscape is installed and configured but not registered.
+        Run `sudo landscape-config` to register, or run `sudo pro disable landscape`
+        """
+        When I run `sudo pro disable landscape` with sudo
+
+        # cleanup
+        Then I reject all pending computers on Landscape
+        Examples: ubuntu release
+            | release | machine_type  |
+            | lunar   | lxd-container |
+
+    @series.lunar
+    @uses.config.machine_type.any
+    @uses.config.machine_type.lxd-container
+    Scenario Outline: Enable Landscape interactively
+        Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
+        When I set the machine token overlay to the following yaml
+        """
+        availableResources:
+        - available: true
+          name: landscape
+        machineTokenInfo:
+          contractInfo:
+            resourceEntitlements:
+              - type: landscape
+                affordances:
+                  series:
+                  - lunar
+                  - mantic
+                directives:
+                  requiredPackages:
+                  - name: landscape-client
+                    removeOnDisable: true
+        """
+        When I attach `contract_token` with sudo and options `--no-auto-enable`
+
+        Then I verify that running `pro enable landscape` `as non-root` exits `1`
+        And I will see the following on stderr:
+        """
+        This command must be run as root (try using sudo).
+        """
+
+        When I run `pro enable landscape` `with sudo` and the following stdin
+        """
+        y
+        $behave_var{machine-name system-under-test}
+        pro-client-qa
+        $behave_var{config landscape_registration_key}
+        $behave_var{config landscape_registration_key}
+
+
+        n
+
+
+        y
+        """
+        Then stdout contains substring:
+        """
+        One moment, checking your subscription first
+        Updating package lists
+        Installing landscape-client
+        Executing `landscape-config`
+        """
+        Then stdout contains substring:
+        """
+        System successfully registered.
+        """
+        Then stdout contains substring
+        """
+        Landscape enabled
+        """
+        When I run `pro status` with sudo
+        Then stdout matches regexp:
+        """
+        landscape +yes +enabled
+        """
+        When I run `pro disable landscape` with sudo
+
+        When I verify that running `pro enable landscape` `with sudo` and the following stdin exits `1`
+        """
+        y
+        $behave_var{machine-name system-under-test}
+        pro-client-qa
+        wrong
+        wrong
+
+
+        n
+
+
+        y
```

Comment:
lol

Reply:
I mean, maybe it would be nice to have a comment nearby explaining what are the prompts we expect here. If this fails at some point because the landscape-config input expectation changed, it is easier to track

Reply:
Good point - I'll add a comment